### ISSUE TITLE
debugging: add a slide about smem tool which generates memory usage

### ISSUE THREAD

**Issue by clementleger** [bootlin/training-materials#179]
https://www.selenic.com/smem/

**Comment by tpetazzoni** [comment]
Don't know if it is that important, but smem has not seen any release since 2013, and the last commit in the Mercurial repo is from 2015. Perhaps it's the kind of tool that doesn't really need any maintenance: it just works and does its job. But I thought it was worth pointing that out.

**Comment by Tropicao** [comment]
I agree on this, I would have a very hard time recommending the tool, since even the documentation seems deprecated now (eg: requesting python2 as a dependency). The only time I mentioned this one was when searching for a tool able to directly output the USS value of a process, but IMHO we should rather give hints about how to retrieve this value from `/proc/<pid>/smaps`
